src/Components/Calculator/Calculator.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import CalculatorKey from "./CalculatorButton";
+import "./Calculator.css";
+
+const Calculator: React.FC = () => {
+    const [isButtonDisabled, setButtonDisabled] = React.useState<boolean>(false);
+    const [isCalcFinished, setCalcFinished] = React.useState<boolean>(true);
+    const [prevDisplay, setPrevDisplay] = React.useState<string>("");
+    const [prevValue, setPrevValue] = React.useState<string>("");
+    const [nextValue, setNextValue] = React.useState<string>("0");
+    const [operator, setOperator] = React.useState<string>("");
```

Comment:
Could you please chage this style `React.useState` to this....

`import {useState} from 'react';`
.....
`const [value, setValue] = useState<string>("");`

in the above case, actually, you don't need to specify the exact type of the variables.
if you write down like this:
`const [value, setValue] = useState("");`
there will be auto-typecasting and the type of value will be set as `string`.